### ISSUE TITLE
Add soil color to chgres_cube

### ISSUE THREAD

**Issue by barlage** [NOAA-EMC/UFS_UTILS#804]
Need to add soil color input data to cold start files. The data are being produced from sfc_climo_gen but not being added to the sfc_data tiles.

**Comment by GeorgeGayno-NOAA** [comment]
@HelinWei-NOAA is adding this field to sfcsub.F. Do we still want this field output to the cold start files?

**Comment by HelinWei-NOAA** [comment]
@GeorgeGayno-NOAA I prefer to have the same way to treat soil color as soil/veg types. If soil/veg types are included in the cold start files, then  let's do the same to soil color.

**Comment by GeorgeGayno-NOAA** [comment]
> @GeorgeGayno-NOAA I prefer to have the same way to treat soil color as soil/veg types. If soil/veg types are included in the cold start files, then let's do the same to soil color.

Yes. soil/veg types are read in from the 'fixed' directory, then output by chgres to the cold start files. If soil color is a static field, it really only needs to be in the cold start file.  sfcsub.F would not need to process it.

**Comment by GeorgeGayno-NOAA** [comment]
@HelinWei-NOAA you recently added processing of this data to the UFS, correct? Do we still need chgres to process it as well? If not, I will close this issue.

**Comment by HelinWei-NOAA** [comment]
@GeorgeGayno-NOAA Yes. We use sfcsub.f to process it for the coldstart. However, like the other soil/veg parameters, it is better to have this function in chgres too.
